x/profile/client/rest/tx.go

```diff
@@ -0,0 +1,114 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/gorilla/mux"
+)
+
+func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/profile/create", createProfileHandler(cliCtx)).Methods("POST")
```

Comment:
I think it's better to change this endpoint to follow the REST indications: 

```suggestion
	r.HandleFunc("/profiles", createProfileHandler(cliCtx)).Methods("POST")
```